test/parallel/test-https-connect-address-family.js

```diff
@@ -20,7 +19,7 @@ function runTest() {
     res.end();
   })).listen(0, '::1', common.mustCall(function() {
     const options = {
-      host: 'localhost',
+      host: ipv6Host,
```

Comment:
given that `// This test that the family option of https.get is honored.` would it be correct to replace this with `'::1'` since this what the server bind to?

Reply:
@refack I think replacing it with a ip would skip the `family` option handling altogether. In net.js:

```js
  // If host is an IP, skip performing a lookup
  var addressType = cares.isIP(host);
  if (addressType) {
    nextTick(self[async_id_symbol], function() {
      if (self.connecting)
        internalConnect(self, host, port, addressType, localAddress, localPort);
    });
    return;
  }
```